### PR TITLE
Adds rtinfo template

### LINKED ISSUE #128: Adds templates for rtinfo
Would be nice to have a template for rtinfo, mostly needed because then we can keep it running after a reboot of 0-os, as 0-os will not be able to persist this.

### PR DESCRIPTION
**Pull request by chrisvdg**
closes #128

### CODE REVIEW

**Comment by chrisvdg** [comment]
Couldn't test yet with rebooting node as it would boot up again using development branch of the templates, the robot would error out because it couldn't find the template.

File: templates/rtinfo/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## template: github.com/threefoldtech/0-templates/node_capacity/0.0.1
+
+### Description
+This template runs an rtinfo client on the local zos host.
+
+### Schema
+
+- `address`: IP address or dns name of the rtinfod server that aggregates the rtinfo.
+- `port`: Port where the rtinfod server is listening for client connections (default: `9930`)
+- `disk`: List of prefixes of disk names to watch (e.g. `["sd"]` to watch `sda`, `sdb`, `sdc`, ... disks)
```
Comment (zaibon) [inline templates/rtinfo/README.md]: use disks since it's a list

File: templates/rtinfo/rtinfo.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    version = "0.0.1"
+    template_name = "rtinfo"
+
+    def __init__(self, name, guid=None, data=None):
+        super().__init__(name=name, guid=guid, data=data)
+        self.add_delete_callback(self.uninstall)
+
+    def validate(self):
+        # validate that rtinfo client is running when installed
+        self._monitor()
```
Comment (zaibon) [inline templates/rtinfo/rtinfo.py]: that's not what the validate method if for. In here you can check the validity of the data in the schema.

File: templates/rtinfo/rtinfo.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    def _monitor(self):
```
Comment (zaibon) [inline templates/rtinfo/rtinfo.py]: Usually the monitor method is a recurring action.
Here again you can make it recurring and check every X minutes that the rtinfo process is still present, if not, restart it.

File: templates/rtinfo/rtinfo.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        except BaseException:
+            pass
+        self.node_sal.client.rtinfo.start(
+            self.data["address"], self.data["port"], self.data["disks"]
+        )
+
+        self.state.set("actions", "install", "ok")
+
+    def uninstall(self):
+        self.node_sal.client.rtinfo.stop(self.data["address"], self.data["port"])
```
Comment (zaibon) [inline templates/rtinfo/rtinfo.py]: probably need to check if is present before trying to stop ? or does stop does nothing when the process doesn't exists ?

Comment (chrisvdg) [reply templates/rtinfo/rtinfo.py]: It doesn't do anything
image

File: templates/rtinfo/README.md (hunk 1 above)
Comment (chrisvdg) [reply templates/rtinfo/README.md]: was using disks, was a typo in the docs

**Pull request merged**